hilti/toolchain/tests/id-base.cc

```diff
@@ -7,10 +7,14 @@
 #include <hilti/autogen/config.h>
 #include <hilti/base/id-base.h>
 
-
 using namespace hilti;
 
-std::string normalizeID(std::string id) { return id; }
+std::optional<std::string> normalizeID(std::string_view id) {
+    if ( id.substr(0, 1) == "%" )
+        return std::string("XXX_") + std::string(id.substr(1));
```

Comment:
Yikes, honestly surprised that a `string_view` cannot be concatenated with a `std::string` and we need to create a throw-away temporary, https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2023/p2591r4.html 😞.